calico-cloud_versioned_docs/version-3.17/get-started/connect/requirements/system-requirements.mdx

```diff
@@ -13,13 +13,14 @@ redirect_from:
 
 Verify that your cluster meets these requirements.
 
-| Required                    | Supported                                                                                                                                                                                                         |
-| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **Platforms**               | All of the open-source [Kubernetes deployments](https://docs.projectcalico.org/getting-started/kubernetes/), except: <br/><br/>- IBM Cloud Kubernetes Service (IKS)<br/>- K3s <br />- Clusters with Windows nodes |
-| **Architecture**            | Only AMD64                                                                                                                                                                                                        |
-| **Kubernetes version**      | - Minimum: v1.23<br/>- Maximum: v1.25<br /><br />If you have a later version than the maximum, contact [Support](https://support.tigera.io/).                                                                     |
-| **CNIs**                    | - Calico CNI v3.15 to v3.24<br />- Amazon VPC CNI<br/>- Azure CNI<br/>- GKE CNI                                                                                                                                                 |
-| **Browsers for Manager UI** | {{prodname}} supports the latest version and "latest minus one" versions of the following browsers:<br/><br/>- Chrome<br/>- Firefox<br/>- Safari                                                                  |
+| Required                    | Supported                                                    |
+| --------------------------- | ------------------------------------------------------------ |
+| **Platforms**               | All of the open-source [Kubernetes deployments](https://docs.projectcalico.org/getting-started/kubernetes/), except: <br/><br/>- OpenShift<br />- IBM Cloud Kubernetes Service (IKS)<br/>- K3s <br />- Clusters with Windows nodes<br />- RKE |
+| **Architecture**            | Only AMD64                                                   |
+| **Kubernetes version**      | - Minimum: v1.23<br/>- Maximum: v1.25<br /><br />If you have a later version than the maximum, contact [Support](https://support.tigera.io/). |
+| **CNIs**                    | - Calico CNI v3.15 to v3.24<br />- Amazon VPC CNI<br/>- Azure CNI<br/>- GKE CNI |
```

Comment:
Hopefully obviously the 2 comments on the non-versioned doc should be reflected here too.

Reply:
Fixed. Thanks!